nvme_host.go

```diff
@@ -0,0 +1,160 @@
+// Copyright © 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goscaleio
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	types "github.com/dell/goscaleio/types/v1"
+)
+
+// NvmeHost defines struct for NvmeHost
+type NvmeHost struct {
+	NvmeHost *types.NvmeHost
+	client   *Client
+}
+
+// NewNvmeHost returns a new NvmeHost
+func NewNvmeHost(client *Client, nvmeHost *types.NvmeHost) *NvmeHost {
+	return &NvmeHost{
+		NvmeHost: nvmeHost,
+		client:   client,
+	}
+}
+
+// GetAllNvmeHosts returns all NvmeHost list
+func (s *System) GetAllNvmeHosts() ([]types.NvmeHost, error) {
+	defer TimeSpent("GetAllNvmeHosts", time.Now())
+
+	path := fmt.Sprintf("/api/instances/System::%v/relationships/Sdc",
```

Comment:
What's the difference between this endpoint and
GET /api/types/Sdc/instances? Looks the return body are totally the same

Reply:
Yes, they look the same and no doc can identify the difference AFAIK. Just follow the existing sdc path.